css/css-counter-styles/armenian/css3-counter-styles-007-ref.html

```diff
@@ -6,7 +6,6 @@
 <link rel='author' title='Richard Ishida' href='mailto:ishida@w3.org'>
 <link rel='help' href='http://www.w3.org/TR/css-counter-styles-3/#simple-numeric'>
 <link rel="reviewer" title="Chris Lilley" href="mailto:chris@w3.org" />
-<meta name='flags' content='font'>
 <meta name="assert" content="list-style-type: armenian produces numbers after 9 items per the spec.">
```

Comment:
This is a ref, and it's curious that the test itself doesn't have the font flag. But anyway, the test passes everywhere, there's nothing to see here:
https://wpt.fyi/results/css/css-counter-styles/armenian/css3-counter-styles-007.html?label=experimental&label=master&aligned